small/small.h

```diff
@@ -81,6 +81,34 @@ struct small_mempool
 	 * pool.
 	 */
 	size_t objsize_min;
+	/**
+	 * Mask for this pool, which shows all pools in the group with an
+	 * object size greater than or equal to the object size of this pool,
+	 * from which there was at least one allocation.
+	 */
+	uint32_t used_pool_mask;
+	/**
+	 * Index of this factor pool in factor pool group with same slab
```

Comment:
There's no more factor pools as I understand.

Reply:
With a low allocation factor, there can be more than 32 pools with the same slab size. We limit this number to 32 pools and, if we have more than 32 pools with same slab size, we make two or more groups of pools with the same slab size